packages/lesswrong/components/users/LoginForm.tsx

```diff
@@ -219,7 +219,6 @@ const LoginFormDefault = ({ startingState = "login", classes }: LoginFormProps)
       {hasOauthSection && <>
         <div className={classes.oAuthComment}>...or continue with</div>
         <div className={classes.oAuthBlock}>
-          <a className={classes.oAuthLink} href={`/auth/facebook?returnTo=${pathname}`}>FACEBOOK</a>
```

Comment:
Noting that `hasOauthSection` is specifically set to `!isEAForum`, rather than to `isLWorAF`, so removing this also removes it for anyone else using the codebase in the future.  (Probably this is fine; if someone has already forked the codebase they're probably careful about pulling in upstream changes.)